cpp/ycm/ClangCompleter/TranslationUnit.cpp

```diff
@@ -38,7 +38,8 @@ namespace {
 unsigned editingOptions() {
   return ( CXTranslationUnit_DetailedPreprocessingRecord |
            clang_defaultEditingTranslationUnitOptions() ) |
-         CXTranslationUnit_IncludeBriefCommentsInCodeCompletion;
+         CXTranslationUnit_IncludeBriefCommentsInCodeCompletion |
+         CXTranslationUnit_Incomplete;
```

Comment:
Could you fix the formatting here while you're at it? The parens are unnecessary and the ordering is inconsistent. Make it `clang_defaultEditingTranslationUnitOptions()` and then the specific options each on their own line.